redis-consumer/Dockerfile

```diff
@@ -3,18 +3,10 @@ FROM python:2.7.11
 MAINTAINER Robert Kaye <rob@metabrainz.org>
 
 # General setup
-RUN apt-get update && apt-get install -y --no-install-recommends python-redis redis-tools vim
```

Comment:
Correct. Since `redis-consumer` doesn't use psql, we don't need to install client here. It can still access the database.